src/extension/TeamCityEventListener.cs

```diff
@@ -37,7 +37,7 @@ namespace NUnit.Engine.Listeners
     // purely documentary since engines prior to 3.4 do not
     // check the EngineVersion property and will try to
     // load this extension anyway.
-    [Extension(Enabled = false, EngineVersion = "3.4")]
+    [Extension(Enabled = false, EngineVersion = "3.5")]
```

Comment:
I suppose that users will install next version of nunit using .zip containing new versions of extensions and a nuget package `NUnit.Runners` will have the same dependencies as .zip. I wish users would used newest versions of NUnit and of TeamCity extensions (if it exists) together (from .zip or nuget). I've made some change in the V2 framework driver extension and have fixed several bugs in the NUnit engine. Integration tests were changed according to those fixes.